### PR TITLE
Monitoring Microservices on OpenShift with Hawkular OpenShift agent

### PR DESCRIPTION
**Pull request by pilhuhn**
This describes monitoring of Obsidian-Java applications

### CODE REVIEW

File: src/main/jbake/content/blog/2017/01/17/obst-hosa.adoc [hunk 1]
```diff
@@ -0,0 +1,269 @@
...
+ifndef::env-github[]
+image::/img/blog/2017/OBST-Pod.png[Pod in Openshift]
+endif::[]
+ifdef::env-github[]
+image::../../../../../assets/img/blog/2017/OBST-Pod.png[Pod in Openshift]
+endif::[]
+
+=== The Hawkular OpenShift Agent
+
+The https://github.com/hawkular/hawkular-openshift-agent[Hawkular OpenShift Agent](Hosa) runs as part of the OpenShift infrastructure inside the _openshift-infra_ namespace on a per node basis, monitoring eligible pods on that node. The GitHub page has pretty good information on how to compile and deploy it (there are also pre-created Docker images available).
```
Comment (jotak) [inline src/main/jbake/content/blog/2017/01/17/obst-hosa.adoc]: Missing space before "(Hosa)"

File: src/main/jbake/content/blog/2017/01/17/obst-hosa.adoc [hunk 2]
```diff
@@ -0,0 +1,269 @@
...
+    endpoints:
+    - type: jolokia # <3>
+      collection_interval: 60s # <4>
+      protocol: "https"
+      tls:
+        skip_certificate_validation: true # <5>
+      port: 8778
+      credentials:
+        username: jolokia
+        password: secret:hosa-secret/password  # <5>
```
Comment (jotak) [inline src/main/jbake/content/blog/2017/01/17/obst-hosa.adoc]: Numbering issue: that's 6 (numbers below are wrong also)

**Review by jotak — COMMENTED** [review]
Nice post!

Some minor issues inline

**Comment by pilhuhn** [comment]
> Nice post!

Thanks!

**Pull request merged**